lib/spack/spack/test/cmd/list.py

```diff
@@ -3,6 +3,10 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import sys
+
+import pytest
```

Comment:
There are no other changes in this file, are these (new) imports required?

Reply:
They are not, I just found those (and a few other new imports) locally a few minutes ago. My local VSCode settings seem to randomly inject imports it thinks I need, I'm looking into disabling this now.